src/frameworks/foundation/ns_bundle.rs

```diff
@@ -124,7 +124,9 @@ pub const CLASSES: ClassExports = objc_classes! {
        withExtension:(id)extension { // NSString *
    msg![env; this URLForResource:name withExtension:extension subdirectory:nil]
 }
-
+- (id)infoDictionary {
+    nil
+}
```

Comment:
If we're going to have this hack, can you add a log message, something like `log!("TODO: [(NSBundle*){:?} infoDictionary] (returning nil)", this);`, so if an app uses this and breaks, there'll be a clue for the problem in the log?